### PR TITLE
Display reCaptcha history on the dashboard (Credits to pokemonkiel.de)

### PR DESCRIPTION
**Pull request by SiteEffect**
## Description

This PR provides a graph of encountered captchas during the past time and serves as a history for captcha activity.
The original code comes from the maintainer of pokemonkiel.de and is 99% of what you see in this PR. All credits go to him, I just serve with this PR to make it public to this project.

## Motivation and Context

Since this website relies on PokemonGo-Map as a core part to provide data, we have to go with the time and watch this project, too. All live Maps encounter reCaptcha at a medium or high rate today and most probably tomorrow. Since this is bounded to costs or user activity to solve them, this the amount of captcha produced ist a key value of interest. In conclusion we want it to be more transparent as well as being able to monitor and track it.

## How Has This Been Tested?

For some time on pokemonkiel.de where it works very fine. Additionally, locally on my fork.
Keep in mind that a working dashboard including a cron task is obligatory for this PR.

## Screenshots:
bildschirmfoto 2017-01-12 um 01 08 21

## WIP
This PR is WIP and might need some additional edits to be merged to the current live branch in terms of code style or general compatibility. Also this could be a toggle-able feature as well since not everyone wants to make their captcha activity public since it can be redirected to the maps expenses (3$ per 1000 reCaptcha for example if 2captcha is used).
If you have any ideas, code or questions to improve or contribute, please feel free to comment.

- [x] update the whole Dashboard layout for nice mobile appearance
- [x] is red the light-right colour?
- [x] make it toggle-able
- [x] provide locales for this PR
- [x] am I an idiot to include unneeded files?

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by SiteEffect** [comment]
Does Scrutinizer like me or is it just checking the very latest changes!?

**Comment by SiteEffect** [comment]
^good boy

**Review by Obihoernchen — CHANGES_REQUESTED** [review]
Looks fine only locale support is missing

**Comment by SiteEffect** [comment]
**To-Do**
- [x] ~~update the whole Dashboard layout for nice mobile appearance~~ (will be addressed in an additional PR)
- [x] is red the light-right colour?
- [x] make it toggle-able
- [x] provide locales for this PR

File: captcha-left.example.php [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-	// allow CORS
-	header('Access-Control-Allow-Origin: *');
-
-
-	// connect to MySQL
-
-	# EDIT ME PLEASE
-
-	$mysql = new mysqli();
-	$mysql->connect('#SYS_DB_HOST#', '#SYS_DB_USER#', '#SYS_DB_PSWD#', '#SYS_DB_NAME#', '3306');
```
Comment (Obihoernchen) [inline captcha-left.example.php]: At least you didn't include your DB credentials xD

Comment (SiteEffect) [reply captcha-left.example.php]: I was very, very careful :-D

**Comment by SiteEffect** [comment]
It's not as nice as the shiny red, but the red signaled me instantly to look at it what is not certainly necessary since those are "only" captcha'd accounts not like banned or so.
Here is the un-shiny yellowish-orangeish you know:
bildschirmfoto 2017-01-13 um 00 35 37
Let me know if you have a nicer colour code.

File: pages/dashboard.page.php [hunk 2]
```diff
@@ -112,16 +112,19 @@
...
+			<div class="col-md-12">
+				<h2 class="sub-title"><strong>reCaptcha</strong> <?= $locales->DASHBOARD_ACTIVITY ?></h2>
 			</div>
 
 			<div class="col-md-12 col-xs-12">
 			
-				<h4>Anzahl Captcha: <small>Letzte 7 Tage</small></h4>
+				<h4><?= $locales->DASHBOARD_CAPTCHA ?> <small><?= $locales->DASHBOARD_LAST7DAYS ?></small></h4>
 			
 				<canvas id="captcha" width="100%" height="25"></canvas>
```
Comment (Obihoernchen) [inline pages/dashboard.page.php]: you forgot to delete this ;)

**Review by Obihoernchen — CHANGES_REQUESTED** [review]

**Review by Obihoernchen — APPROVED** [review]

**Comment by SiteEffect** [comment]
Thank you, will be addressed!
What do you mean with ``-w`` btw?

**Comment by Obihoernchen** [comment]
Ouh shit I missed this. Yes we have to change this to $time_offset

**Pull request merged**